### ISSUE TITLE
关于代码中nnPU的一些疑问

### ISSUE THREAD

**Issue by Wongcheukwai** [VITA-Group/Self-PU#3]
在Self-PU/utils/util.py的第238行， return R_p - BETA, -gamma*R_n 这里是不是有错鸭，因为这个意思是，如果Rn>0的话，应该return Rp+0吧？因为在主程序里你返回的参数是， _, loss = criterion(output1, Y)，按照这样的话，你返回的loss是Rn

**Comment by chanyh0** [comment]
We do need to return \gamma * Rn here. Please see Algorithm 1 in https://arxiv.org/pdf/1703.00593.pdf. 
Also, you can refer to https://github.com/kiryor/nnPUlearning/blob/a522c0edd22c54b627eb131e960584950eae5330/pu_loss.py#L51 for help. We adapted their code.
